Source/Scene/Camera.js

```diff
@@ -2841,13 +2819,20 @@ define([
     Camera.clone = function(camera, result) {
         if (!defined(result)) {
             result = new Camera(camera._scene);
+            result.frustum = camera.frustum.clone(); // Allocate a new frustum of the correct type
         }
 
         Cartesian3.clone(camera.position, result.position);
         Cartesian3.clone(camera.direction, result.direction);
         Cartesian3.clone(camera.up, result.up);
         Cartesian3.clone(camera.right, result.right);
         Matrix4.clone(camera._transform, result.transform);
+        result._transformChanged = true;
+
+        // Clone frustum only if it is the same type
+        if (defined(camera.frustum.left) === defined(result.frustum.left)) {
```

Comment:
Yes that was the reason. The shadows code doesn't depend on this check though. Should I remove this, or expand on the comment?